demo/ffmpeg-to-shaka-packager/dashTestContent.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+port=8080
+# create a unique directory
+dir=$(date '+%m-%d-%y-%T')
+vid='BigBuckBunny'
```

Comment:
Same as last PR, use uppercase var names for constants.
https://google.github.io/styleguide/shellguide.html#naming-conventions

For `dir`, it might be good to clarify what this directory is used for.
Same for `vid`. Is this an input or an output video file? It looks like an output filename, but is unclear from this here without reading the rest of the code.